shared/uavobjectdefinition/hwbrainre1.xml

```diff
@@ -54,6 +61,24 @@
 		</options>
 	</field>
 
+	<field name="MultiPortSerial2" units="function" type="enum" elements="1" parent="HwShared.PortTypes" defaultvalue="Disabled">
+		<options>
+			<option>Disabled</option>
```

Comment:
Exactly. IMO it would be confusing to have them and lead to user error. The only scenario that I can see where this would be useful would be if the other serial port is damaged.